ui/src/app/components/breadcrumb.tsx

```diff
@@ -167,10 +167,13 @@ export const Breadcrumb = fp.flow(
       if (!prevProps.workspace && this.props.workspace &&
         this.props.workspace.billingStatus === BillingStatus.INACTIVE) {
         this.setState({showInvalidBillingBanner: true});
-      }
-
-      if (prevProps.workspace && !this.props.workspace) {
+      } else if (prevProps.workspace && !this.props.workspace) {
```

Comment:
[No action] I haven't come up against the breadcrumb code before, so it's a little surprising to see some very workspace & billing specific logic showing up in a component that otherwise seems to be very generic. Could you imagine (not for this PR, or not anytime soon) a better home for this? Or is there a a strong reason that this logic belongs in this component?

Reply:
Definitely a bit of a hack. It's mainly being used because it was the best React component I could find that is loaded on every workspace page.

I had initially tried to put it in `workspace-wrapper` but that is still in angular.